labelbox/data/serialization/ndjson/classification.py

```diff
@@ -28,8 +33,16 @@ def dict(self, *args, **kwargs):
         res = super().dict(*args, **kwargs)
         if 'name' in res and res['name'] is None:
             res.pop('name')
+        #NOTE some glitch exists and not matter what I tried with using our CamelCaseMixin or by_alias is the CUID field defition,
```

Comment:
I tested this myself, and looks like `schema_id` is converted successfully to camelCase when we run the serializer. How did you come across this glitch?

Reply:
comment out this line and run, for example `pytest tests/data/serialization/ndjson/test_checklist.py -k "test_serialization_with_classification_double_nested_2"`
This must be some pydantic weirdness on our side or their side, I just did not have enough time to dig into it

Reply:
Ran the test and it works for me. The schema id is successfully converted to camelCase
```RES: {'confidence': 0.42, 'name': 'third_subchk_answer', 'schemaId': None, 'classifications': []}```

Reply:
yep I have fixed it :)